junit-platform-engine/src/main/java/org/junit/platform/engine/reporting/ReportEntry.java

```diff
@@ -83,7 +83,7 @@ private void add(String key, String value) {
 	 *
 	 * @return a copy of the map of key-value pairs; never {@code null}
 	 */
-	public final Map<String, String> getKeyValuePairs() {
+	public Map<String, String> getKeyValuePairs() {
```

Comment:
Apologies if I'm missing something: does this address a particular issue, as in does something need to override these methods as I can't see how this relates to the description or overview in the PR?

Thanks

Reply:
Can't remember why I did that tbh, I think it was related to the class being final, so the IDE was complaining that the functions within are final. Stupid change, shall remove 🤦

Reply:
Re-added it back in, along with the getTimestamp() function